go/staking/tests/tester.go

```diff
@@ -179,8 +179,14 @@ func testTransfer(t *testing.T, state *stakingTestsState, backend api.Backend, c
 	err = consensusAPI.SignAndSubmitTx(context.Background(), consensus, srcSigner, tx)
 	require.NoError(err, "Transfer")
 
+TransferWaitLoop:
 	select {
 	case ev := <-ch:
+		if ev.From.Equal(api.CommonPoolAccountID) || ev.To.Equal(api.CommonPoolAccountID) || ev.From.Equal(api.FeeAccumulatorAccountID) || ev.To.Equal(api.FeeAccumulatorAccountID) {
+			// This is not the transfer we're looking for.
+			goto TransferWaitLoop
```

Comment:
Use a `for` and continue? Also could you instead test that these fee-related events are there instead of just skipping them? Same below.

Reply:
Done!